easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-3.0.0-gompi-2022a.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'CMakeMake'
+
+name = 'ncbi-vdb'
+version = '3.0.0'
+
+homepage = 'https://github.com/ncbi/ncbi-vdb'
+description = """The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
+ using data in the INSDC Sequence Read Archives."""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+
+github_account = 'ncbi'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+patches = [
+    'ncbi-vdb-2.10.7_fix-LD_LIBRARY_PATH.patch',
+    'ncbi-vdb-cstdlib.patch',
+]
+checksums = [
+    '154317ef265104861fe8d3d2e439939ae98f33b1e28da3c45f32ae8534dbfad7',  # ncbi-vdb-3.0.0.tar.gz
+    'e8f22dbd0c2e564e296bafdf76ba0e0e2da0d13e22be5aaf322135e5f26eb133',  # ncbi-vdb-2.10.7_fix-LD_LIBRARY_PATH.patch
+    'db3d563262ca9b14e7b9a94a0be6683a9eef41a498c9c064aa05c2a6785f1655',  # ncbi-vdb-cstdlib.patch
+]
+
+builddependencies = [
+    ('Perl', '5.34.1', '-minimal'),
+    ('Python', '3.10.4', '-bare'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('file', '5.43'),  # provides libmagic
+    ('HDF5', '1.10.8'),  # version 1.12.x has changes to API and is not compatible
+    ('libxml2', '2.9.13'),
+    ('bzip2', '1.0.8'),
+]
+
+# add addtional libraries needed to statically link HDF5 from EB
+preconfigopts = "sed -i 's/-lhdf5 -Wl,-Bdynamic/-lhdf5 -Wl,-Bdynamic -lmpi -lsz/'"
+preconfigopts += " %(builddir)s/%(namelower)s-%(version)s/setup/konfigure.perl &&"
+
+configopts = ''
+configopts += ' -DHDF5_LIBDIR=$EBROOTHDF5/lib -DXML2_LIBDIR=$EBROOTLIBXML2/lib'
+configopts += ' -DHDF5_INCDIR=$EBROOTHDF5/include -DXML2_INCDIR=$EBROOTLIBXML2/include'
+
```

Comment:
I'm assuming whatever happened [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/7c4cd2735f43c34c93355f01f9d5a2f93ac218ef/easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.10.7-gompi-2020a.eb#L35) in prior versions isn't needed anymore?

Reply:
I commented earlier on dropping the ngs library dependency and file checks, but the other part is that ncbi-vdb is switching to CMAKE, hence the use of -D instead of --with.

Reply:
Ok, together with your comment below, that's all clear.